editor/src/utils/utils.ts

```diff
@@ -935,6 +935,18 @@ function deduplicateBy<T>(key: (t: T) => string, ts: Array<T>): Array<T> {
   return results
 }
 
+function groupBy<Element>(
```

Comment:
Can you move this into [`array-utils.ts`](https://github.com/concrete-utopia/utopia/blob/master/editor/src/core/shared/array-utils.ts) please

Reply:
done!